### PR TITLE
Gamelab: Show coordinates below visualization

### PR DESCRIPTION
**Pull request by islemaster**
out

Finishes [axof: 482]

We now display the mouse coordinates (in game-space) below the Run/Reset button, even when the game is running (but not in share mode).

### CODE REVIEW

**Comment by bcjordan** [comment]
Minor visual nit / idea re: thrashing of the y coordinate location, wonder if something like a `min-width` span/div to keep the `x` coordinate's width changes from affecting the `y` one might help

File: apps/src/gamelab/GameLabVisualizationColumn.jsx [hunk 1]
```diff
@@ -14,49 +14,87 @@ import TooltipOverlay, {coordinatesProvider} from '../templates/TooltipOverlay';
...
-      <BelowVisualization instructionsInTopPane={props.instructionsInTopPane}/>
-    </span>
-  );
-};
-
-GameLabVisualizationColumn.propTypes = {
-  finishButton: React.PropTypes.bool.isRequired,
-  instructionsInTopPane: React.PropTypes.bool.isRequired
-};
+var GameLabVisualizationColumn = React.createClass({
```
Comment (bcjordan) [inline apps/src/gamelab/GameLabVisualizationColumn.jsx:17]: (For my own understanding) Converted to a full React class here due to addition of `this.state`?

**Comment by bcjordan** [comment]
[?w=0](https://github.com/code-dot-org/code-dot-org/pull/11529/files?w=0) recommended!

File: apps/src/gamelab/GameLabVisualizationColumn.jsx [hunk 2]
```diff
@@ -14,49 +14,87 @@ import TooltipOverlay, {coordinatesProvider} from '../templates/TooltipOverlay';
...
-var GameLabVisualizationColumn = function (props) {
-  var divGameLabStyle = {
-    width: GAME_WIDTH,
-    height: GAME_HEIGHT
-  };
-  return (
-    <span>
-      <ProtectedVisualizationDiv>
-        <div id="divGameLab" style={divGameLabStyle} tabIndex="1">
-        </div>
```
Comment (bcjordan) [inline apps/src/gamelab/GameLabVisualizationColumn.jsx:26]: Initially read this as storing (data-wise) the coordinates in app-space, maybe `Cache app-space mouse coordinates`

File: apps/src/gamelab/GameLabVisualizationColumn.jsx (hunk 1 above)
Comment (islemaster) [reply apps/src/gamelab/GameLabVisualizationColumn.jsx:17]: Correct!

File: apps/src/gamelab/GameLabVisualizationColumn.jsx (hunk 2 above)
Comment (islemaster) [reply apps/src/gamelab/GameLabVisualizationColumn.jsx:26]: Updated.

**Comment by islemaster** [comment]
Added a min-width to the x coordinate per suggestion.

**Comment by bcjordan** [comment]
👍

**Comment by caleybrock** [comment]
Commit 142005008293d56e9ea0b7a5650e87d3186d1ed3 creates `VM973:27 Warning: Component's children should not be mutated.`

**Pull request merged**